### PR TITLE
internal endpoints swagger additional endpoints

### PR DESCRIPTION
**Pull request by supersven**
This adds internal Servant endpoints to the Swagger docs for internal endpoints. (`brig` and `cannon` endpoints were already there.)

To have them available I had to move some types from `galley-types` to `wire-api`. `spar` and `galley` internal API definitions haven been moved there, too.

Follow up to https://github.com/wireapp/wire-server/pull/3003

## Rendered
image

## Checklist

 - [x] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [x] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

File: services/brig/docs/swagger-internal-endpoints.md [hunk 1]
```diff
@@ -5,3 +5,8 @@ between services and site operators for forensics.
 
 Request execution does not work as Swagger expects a single target host whereas
 these endpoints are served by multiple hosts.
+
+Displayed paths are prefixed by the service name (e.g. `/<brig>`). This is
+necessary because Swagger does not expect more than one endpoint for a given
+path and method. Though, some paths (e.g. `/i/status`) exist on more than one
+service. To use the a path, remove the prefix.
```
Comment (battermann) [inline services/brig/docs/swagger-internal-endpoints.md:12]: Alternatively, we could expose the internal swagger UI for each service on their own page?

Is it possible to use the swagger UI to call those endpoints, the way it is implemented now?

**Review by battermann — APPROVED** [review]

File: services/brig/docs/swagger-internal-endpoints.md (hunk 1 above)
Comment (supersven) [reply services/brig/docs/swagger-internal-endpoints.md:12]: Thanks for the inspiration! I was more focused on the overview over all endpoints. However, being able to use the endpoints is pretty appealing, too.

As this should be possible with a `kubectl port-forward`, I'll give it a shot.

Comment (fisx) [reply services/brig/docs/swagger-internal-endpoints.md:12]: maybe different PR, though?

Comment (supersven) [reply services/brig/docs/swagger-internal-endpoints.md:12]: > maybe different PR, though?

@fisx , Sounds good :+1:

**Pull request merged**